src/coreclr/tests/dir.traversal.targets

```diff
@@ -14,7 +14,7 @@
              Properties="Dummy=%(Identity);DefaultBuildAllTarget=$(DefaultBuildAllTarget);BuildAllProjects=true"
              ContinueOnError="ErrorAndContinue" />
 
-    <MSBuild Targets="$(DefaultBuildAllTarget)"
+    <MSBuild Targets="$(ç)"
```

Comment:
Is this a typo or a GitHub quirk?

Reply:
Typo, although I am not sure how it was like.. .working with that typo there?